Specifications/Tool/com.apple.build-tools.strip.xcspec

```diff
@@ -0,0 +1,22 @@
+/**
+ Copyright (c) 2015-present, Facebook, Inc.
+ All rights reserved.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree. An additional grant
+ of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+{
+    Type = Tool;
+    Identifier = com.apple.build-tools.strip;
+    Name = "Strip";
+
+    CommandLine = "strip [options] [input]";
+
+    RuleName = "Strip $(InputFile)";
+
+    Outputs = (
+        "$(OutputPath)",
+    );
+}
```

Comment:
No options here. I think `STRIP_STYLE` is the only one (?), so should be easy enough to flesh out (or add a TODO).